app/search_builders/hyrax/dashboard/collections_search_builder.rb

```diff
@@ -9,7 +9,7 @@ class CollectionsSearchBuilder < Hyrax::CollectionSearchBuilder
 
       # This overrides the models in FilterByType
       def models
-        [::AdminSet, ::Collection, Hyrax.config.collection_model.safe_constantize].uniq.compact
+        [::AdminSet, ::Collection, Hyrax.config.collection_class].uniq.compact
```

Comment:
This is not part of the fix.  It updated the code for consistency `collection_class` is already constantized in `Hyrax.config`